mlir/lib/ExecutionEngine/ROCm/rocm-runtime-wrappers.cpp

```diff
@@ -314,6 +329,129 @@ extern "C" void mgpuMemCopyInt32(int32_t *sourceAllocated,
             static_cast<hipMemcpyKind>(copyDirection));
 }
 
+extern "C" StridedMemRefType<int8_t, 5>
+mgpuMemAlloc5DInt8(int8_t *allocated, int8_t *aligned, int64_t offset,
+                   int64_t size0, int64_t size1, int64_t size2, int64_t size3,
+                   int64_t size4, int64_t stride0, int64_t stride1,
+                   int64_t stride2, int64_t stride3, int64_t stride4) {
```

Comment:
This is fine for now but we should probably just have a single alloc func and insert casts from it.

Reply:
@yiqian1 can you file a ticket for that. thx